docs/reference/reference.rst

```diff
@@ -0,0 +1,59 @@
+Reference
+=========
+
+.. contents::
+    :depth: 3
+    :local:
+
+Programmatically accessing the graph
+------------------------------------
+
+.. autoclass:: qmxgraph.api.QmxGraphApi
+    :members:
+
+Qt widget
+-----------------
+
+.. autoclass:: qmxgraph.widget.QmxGraph
+    :members:
+
+.. autoclass:: qmxgraph.widget.EventsBridge
+    :members:
+
+MIME (for drag & drop into the widget)
+--------------------------------------
+
+.. automodule:: qmxgraph.mime
+    :members:
+
+constants
```

Comment:
Is this synced with https://prusse-qmxgraph-fork.readthedocs.io/en/initial-rtd/installation.html ? Because some sections are not visible (I believe constants doesn't have anything indeed, but `GraphOptions` and `GraphStyles` have some docstring).

Reply:
I will check those.
Locally it was displaying... Maybe I got something extra installed locally while testing out sphinx.

Reply:
The problem was with the non declared dependency to PyQt.
My guess is that the Qt dependency has been left out because qmx's API can be used without the Qt infrastructure (in a browser), we actually test most of the javascript code without Qt.

After that was fixed I did need to conditionally import the resources... I will like to avoid that maybe someone more "pip" savvy can offer a better approach to this mess.

Reply:
> After that was fixed I did need to conditionally import the resources.

~~What do you mean?~~


Nevermind, I see what you mean.

`pip` cannot really help you here. I think that approach is fine.